lib/ococo/events/orders_order_fill.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = async (instance = {}) => {
+  const { state = {}, h = {} } = instance
+  const { updateState, emitSelf } = h
+  const { initialOrderFilled } = state
+
+  if (initialOrderFilled) {
+    await emitSelf('exec:stop')
+  } else {
+    await updateState(instance, { initialOrderFilled: true })
```

Comment:
Maybe we should add:

```
if (order.amount < DUST)
```

Or did you want the OCO to trigger on a partial fill?